Jenkinsfile

```
@@ -586,63 +586,63 @@ pipeline {
                             println('------------------------------------------------------------')
                             sequences = [
                                 [description: 'run_if_pr=true',
-                                 kwargs: [tags: 'pr', pragma_suffix: '-hw-medium', distro: null, run_if_pr: true],
+                                 kwargs: [tags: 'pr', pragma_suffix: '-hw-medium', distro: null, run_if_pr: true, run_if_landing=true],
```

I think you want:

```suggestion
                                 kwargs: [tags: 'pr', pragma_suffix: '-hw-medium', distro: null, run_if_pr: true, run_if_landing: true],
```

Many below need the same correction.

---

Yes, thanks.